vignettes/Model.Rmd

```diff
@@ -10,90 +12,365 @@ vignette: >
+
+## Human Biology
+
+The human variables are documented in [R/variables.R](https://github.com/mrc-ide/malariasimulation/blob/master/R/variables.R).
+
+The functions governing the human flow of infection and immunity processes are described in the following files:
+
+1.  [R/human_infection.R](https://github.com/mrc-ide/malariasimulation/blob/master/R/human_infection.R)
+2.  [R/disease_progression.R](https://github.com/mrc-ide/malariasimulation/blob/master/R/disease_progression.R)
+3.  [R/mortality_processes.R](https://github.com/mrc-ide/malariasimulation/blob/master/R/mortality_processes.R)
+
+<img src="Model_human.svg" style="display:block; margin: 0 auto; border-width: 0; width: 100%"/>
+
+### States
+
+Modelled human states are Susceptible (*S*), Treated (*Tr*), Clinical disease (*D*), Asymptomatic infection (*A*) and Sub-patent infection (*U*).
+
+### Parameters
+
+Parameters shown on the infographic include:
+
+-   $\Lambda_i$: the age-specific (*i*) force of infection
+-   $\phi_i$: the age-specific (*i*) probability of clinical disease
+-   $f_T$: the probability of receiving treatment (or force of treatment)
+
+with the rates of recovery:
+
+-   $r_D$: from clinical disease to asymptomatic infection
+-   $r_A$: from asymptomatic infection to sub-patent infection
+-   $r_U$: from sub-patent infection to complete recovery
+-   $r_T$: from treated clinical disease to complete recovery
+
+Superinfection may occur in individuals with asymptomatic or sub-patent infections at the same rates as standard infection (dashed arrows).
+
+The force of infection ($\Lambda_i$) is impacted by pre-erythrocytic immunity, mosquito biting rate and population size and level of infectivity (specific details can be found in the references below). All default parameters can be found in the documentation for the function `get_parameters()`. Please also note that while the infographic above displays rate parameters, the parameter list uses delays durations, e.g. the rate $r_D$ is the inverse of the delay from state *D* to *A* (`dr` in the model):
+
+$$r_D=dr^{-1}$$
+
+To maintain a constant population size during simulations, the birth rate of new susceptible individuals is set to be equal to the overall death rate.
+
+## Mosquito Biology
+
+The functions governing mosquito biological processes and dynamics are spread out between the following files:
+
+1.  [src/adult_mosquito_eqs.cpp](https://github.com/mrc-ide/malariasimulation/blob/master/src/adult_mosquito.cpp)
+2.  [src/aquatic_mosquito_eqs.cpp](https://github.com/mrc-ide/malariasimulation/blob/master/src/aquatic_mosquito.cpp)
+3.  [src/mosquito_biology.cpp](https://github.com/mrc-ide/malariasimulation/blob/master/src/mosquito_biology.cpp)
+4.  [R/mosquito_biology.R](https://github.com/mrc-ide/malariasimulation/blob/master/R/mosquito_biology.R)
+
+<img src="Model_mos.svg" style="display:block; margin: 0 auto; border-width: 0; width: 80%"/>
```

Comment:
I think this "NonExistent" is an underlying model implementation part that doesn't need to be shown the user. The more standard way to represent mortality in these flow diagrams would to have the death arrows just going into thin air.

It currently makes "nonExistent" the centre piece of the diagrame which I found confusing, so can we remove it? Otherwise is really nice 👍

Reply:
I've redone this, taking out the NonExistent and replaced it with a single direction model, which I think makes more sense now.